src/core/core.scale.js

```diff
@@ -495,10 +495,10 @@ module.exports = function(Chart) {
 			var min = me.min;
 			var max = me.max;
 
-			return me.beginAtZero ? 0:
-				min < 0 && max < 0? max :
-				min > 0 && max > 0? min :
-				0;
+			return me.beginAtZero ? 0 :
```

Comment:
I thought so at first too, but then I realized that it more clearly communicates that each line is a continuation of the past and each new line is an else statement of the previous.

In general, I think it's a good rule that continued lines should be indented from the previous. I'm not sure we'd want to disable that entirely because of just one place where we might not like the effect